pycbc/events/stat.py

```diff
@@ -1555,6 +1563,12 @@ def rank_stat_coinc(self, s, slide, step, to_shift,
 
         # First get signal PDF logr_s
         stat = {ifo: st for ifo, st in s}
+
+        try:
+            self.curr_mchirp = kwargs['mchirp']
+        except KeyError:
+            logging.info("Kwargs doesn't contain mchirp")
```

Comment:
again, this seems like it should be an actual error

Reply:
(and the message doesnt seem that useful)

Reply:
Will change it to an actual raised error. The error message theoretically shouldn't ever be met so I can try and think of a more useful message to display